app.js

```diff
@@ -0,0 +1,23 @@
+const body = document.querySelector("body");
+const initialContent = document.querySelector(".initial-content");
+const mainKeycode = document.querySelector(".keycode-info");
+const keyCard = document.querySelector(".key-card");
+const codeCard = document.querySelector(".code-card");
+const whichCard = document.querySelector(".which-card");
+
+document.addEventListener("keydown", (event) => {
+   initialContent.style.display = "none";
+   //console.log(event);
+   let keyNumber = event.which;
```

Comment:
Do you need to store these values in variables?

Reply:
No I do not - Thank you :-))